test/api_client_test.rb

```diff
@@ -44,25 +44,37 @@ def test_logger
   end
 
   def test_invalid_grant_error
-    assert_authentication_error('error' => 'invalid_grant')
+    body = {
+      "message":"Error validating grant. Your authorization code or refresh token may be expired or it was already used.",
+      "error":"invalid_grant",
+      "status":400,
+      "cause":[]
+    }
+    assert_authentication_error(body)
   end
 
   def test_forbidden_error
-    assert_authentication_error('error' => 'forbidden')
+    body = { "message":"forbidden", "error":"forbidden", "status":400, "cause":[] }
+    assert_authentication_error(body)
   end
 
   def test_message_error
-    assert_token_error('message' => 'Malformed access_token')
+    body = { "message": "Malformed access_token: test","error": "bad_request", "status": 400, "cause":[]}
+    assert_token_error(body)
   end
 
   def test_invalid_token_error
-    assert_token_error('error' => 'invalid_token')
+    body = { "message": "invalid_token", "error": "not_found", "status": 401, "cause":[] }
```

Comment:
I took the error responses from recorded vcr files, I don't have a vcr for the forbidden error, but we can update it in case the error message is different.